src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -8,7 +8,7 @@ function getPrimaryContact(data) {
   // treated as the primary contact.
   return {
     name: data?.applicantName ?? false,
-    email: false, // We don't collect email
+    email: data?.applicantEmail,
```

Comment:
This unit test is failing - I think this line just needs to be `data?.applicantEmail ?? false,` and that should fix it. Thanks!

![Screenshot 2024-12-13 at 08 03 42](https://github.com/user-attachments/assets/c90c5e80-7a00-4adf-beef-f90b9e0235fb)